examples/default.yaml

```diff
@@ -215,6 +215,18 @@ video:
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
   # 🟢 Builtin default: "none"
   display: null
+  # VNC (Virtual Network Computing) is a platform-independent graphical
+  # desktop-sharing system that uses the Remote Frame Buffer protocol (RFB)
+  vnc:
+    # VNC display, e.g., "127.0.0.1:0", "host:d", "unix:path", "none"
+    # By convention the TCP port is 5900+d, connections from any host.
+    display: null
+    # VNC websocket, e.g. "127.0.0.1:5700", "host:port"
+    # By convention the TCP port is 5700+d, connections from any host.
+    websocket: null
+    # VNC password, e.g. "on", "off"
+    # Use QMP to set password: `change-vnc-password password=secret`
+    password: null
```

Comment:
Password is necessary and should be autogenerated and saved as a text file under the instance dir

Reply:
https://pkg.go.dev/github.com/digitalocean/go-qemu/qmp/raw#Monitor.ChangeVNCPassword